File: #1887.reduction-operations-to-make-the-array-elements-equal.cpp
class Solution {
public:
    int reductionOperations(vector<int>& nums) {

         sort(nums.begin() , nums.end());
         
         int size = nums.size();
         int cnt = 0; 
         int result = 0;
         for (int i = 1 ; i < size ; i++) {
            result += (nums[i] != nums[i - 1]) ? ++cnt : cnt;  
         }
         return result; 
    }
};

File: #1887.reduction-operations-to-make-the-array-elements-equal.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var reductionOperations = function(nums) {
    let result = 0;
    let num = nums.sort((left, right) => right - left)[0];
    for (let i = 0; i < nums.length; i++) {
        if (num != nums[i]) {
            result += i;
            num = nums[i];
        }
    }
    return result;
};
